features/dd-sdk-android-trace/src/test/kotlin/com/datadog/android/trace/internal/domain/event/OtelDdSpanToSpanEventMapperTest.kt

```diff
@@ -0,0 +1,202 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.trace.internal.domain.event
+
+import com.datadog.android.api.context.DatadogContext
+import com.datadog.android.core.internal.utils.toHexString
+import com.datadog.android.log.LogAttributes
+import com.datadog.android.trace.assertj.SpanEventAssert.Companion.assertThat
+import com.datadog.android.utils.forge.Configurator
+import com.datadog.trace.api.DD128bTraceId
+import com.datadog.trace.core.DDSpan
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.annotation.BoolForgery
+import fr.xgouchet.elmyr.annotation.Forgery
+import fr.xgouchet.elmyr.annotation.StringForgery
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.kotlin.whenever
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(MockitoExtension::class), ExtendWith(ForgeExtension::class)
+)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@ForgeConfiguration(Configurator::class)
+internal class OtelDdSpanToSpanEventMapperTest {
+
+    lateinit var testedMapper: OtelDdSpanToSpanEventMapper
+
+    @Forgery
+    lateinit var fakeDatadogContext: DatadogContext
+
+    @BoolForgery
+    var fakeNetworkInfoEnabled: Boolean = false
+
+    // region Tests
+
+    @BeforeEach
+    fun `set up`() {
+        testedMapper = OtelDdSpanToSpanEventMapper(fakeNetworkInfoEnabled)
+    }
+
+    @Test
+    fun `M map a DdSpan to a SpanEvent W map()`(
+        @Forgery fakeSpan: DDSpan
+    ) {
+        // Given
+        val expectedMeta = fakeSpan.baggage + fakeSpan.tags.map {
+            it.key to it.value.toString()
+        }
+        val expectedMetrics = fakeSpan.expectedMetrics()
+
+        // When
+        val event = testedMapper.map(fakeDatadogContext, fakeSpan)
+
+        // Then
+        assertThat(event).hasSpanId(fakeSpan.spanId.toHexString())
```

Comment:
it seems we need to wrap lines of this assertion block, each `hasXXX` should be on the new line for readability. Same applies for other tests in this file.